pkg/apis/eksctl.io/v1alpha5/assets/schema.json

```diff
@@ -150,25 +150,35 @@
     "ClusterIAM": {
       "properties": {
         "fargatePodExecutionRoleARN": {
-          "type": "string"
+          "type": "string",
+          "description": "role used by pods to access AWS APIs. This role is added to the Kubernetes RBAC for authorization. See [Pod Execution Role](https://docs.aws.amazon.com/eks/latest/userguide/pod-execution-role.html)",
+          "x-intellij-html-description": "role used by pods to access AWS APIs. This role is added to the Kubernetes RBAC for authorization. See <a href=\"https://docs.aws.amazon.com/eks/latest/userguide/pod-execution-role.html\">Pod Execution Role</a>"
         },
         "fargatePodExecutionRolePermissionsBoundary": {
-          "type": "string"
+          "type": "string",
+          "description": "permissions Boundary for the FargatePodExecutionRole",
```

Comment:
There's a lot of these keys all over the rest of the schema, where there's no way to summarize the key or any docs to explain it so it ends up being a reformulation of the key name. I think we should just leave out the description in that case. WDYT?

Reply:
Not a reason not to merge the PR actually. If you agree though, I'll go through and make a new PR where I remove all comments like that.